lib/HttpClient.js

```diff
@@ -259,7 +257,7 @@ function rawRequest(opts, cb) {
                 if (req) {
                     req.abort();
                     process.nextTick(function () {
-                        req.emit('result', err, null);
+                        emitResult(err, req, null);
```

Comment:
are we actually emitting the after event in the base http client? I'm not seeing it here.

Reply:
Yes, it's not implemented. As I note in the original description, HttpClient consumes the streams directly via the `req.on('result')` event. The inherited String/JSONClient consume that same event in order to parse results before emitting the `after` event. Thus, I'm not sure what `after` means semantically in the lower level HttpClient. I don't feel strongly about implementing it there since it's so low level - but can open for discussion.

Reply:
If you can implement it the low-level client -- it might be a good idea since it means that this event is emitted for all 3 clients? Also, are we at risk of http client changes breaking the upstack clients if it's not implemented in the base class?

Reply:
Any suggestions for what `after` means in the HttpClient? Since the lower level client is interacting with streams directly, the consumer actually owns the request lifecycle.